MDANSE/Src/MDANSE/Framework/Configurators/AtomTransmutationConfigurator.py

```diff
@@ -34,31 +34,30 @@ def __init__(self, trajectory: Trajectory) -> None:
         system : ChemicalSystem
             The chemical system object.
         """
-        self.selector = Selector(trajectory)
+        self.selector = ReusableSelection()
         self._original_map = {}
         for number, element in enumerate(trajectory.chemical_system.atom_list):
             self._original_map[number] = element
         self._new_map = {}
+        self._current_trajectory = trajectory
 
-    def apply_transmutation(
-        self, selection_dict: dict[str, Union[bool, dict]], symbol: str
-    ) -> None:
+    def apply_transmutation(self, selection_string: str, symbol: str) -> None:
```

Comment:
I realise this is existing API, but I'm not sure what "transmutation" is being applied here? It doesn't seem to be changing anything other than the selection? The docstring isn't helping me either.